src/states/vermont/main.py

```diff
@@ -0,0 +1,72 @@
+import os
+import os.path
+import time
+from ediblepickle import checkpoint
+from selenium import webdriver
+import pandas as pd
+from common import key_namer, work_dir, to_list
+
+BASE_URL = 'https://sos.vermont.gov/elections/town-clerks/'
```

Comment:
So `wget https://sos.vermont.gov/media/fyvdm1j0/clerkcontactinfoexcel.xlsx` seems to work.  Are you afraid that the link will change?

Reply:
I took this approach because `cache_request` returns an html page with js instead of the expected Excel file. Selenium clicks the link, gets through the js, and downloads the file through the browser. Protection against the url changing is a side benefit.

I just tried `wget` (through python) and verified it has the same problem (gets html +js instead of the Excel file).